### PR TITLE
Combine ZStackEdit and TrackEdit classes to ensure consistent `cell_info` contents

### PR DESCRIPTION
**Pull request by tddough98**
Since we've added tracking to the current build of DeepCell Label, we've been creating tracking metadata for all projects so that any project could use the tracking code and UI. However, the edit route still distinguished between tracking and zstack projects using the extension of the file used to create the project. Only files ending with .trk or .trks are edited with TrackEdit, and all other files are edited with ZStackEdit. As each class implements its own `add_cell_info` and `del_cell_info`, editing with the ZStackEdit would not maintain the tracking metadata as expected.

Issue #280 revealed this issue, as a project with no initial labels would have an empty `cell_info` dict, and once we start adding labels using the ZStackEdit class, the `cell_info` would not have the expected fields. The route that serves `cell_info` expects these fields to be present, and serves an error instead and causing NaN to be appear the selected label.

Here the TrackEdit and ZStackEdit classes are unified into a single Edit class so that all projects are both initialized and updated with the same consistent set of field in its `cell_info` dict. I also removed actions that are not currently used.

### CODE REVIEW

File: visualizer/src/service/apiMachine.js [hunk 1]
```diff
@@ -32,19 +33,26 @@ function redo(context, event) {
...
+  const format = new URLSearchParams(window.location.search).get('track') ? 'trk' : 'npz';
+  const promise = axios.get('/api/download', {
+    params: { id: projectId, format: format },
+    responseType: 'blob',
+  });
+  promise.then(response => console.log(response));
   const filename = promise.then(response => {
     const regex = /filename=(.*)$/;
-    const header = response.headers.get('content-disposition');
+    const header = response.headers['content-disposition'];
```
Comment (willgraf) [inline visualizer/src/service/apiMachine.js:55]: Why the switch from `fetch` to `axios`?

File: deepcell_label/utils.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            parent = info['parent']
+            info['frame_div_parent'] = new_info[parent]['frame_div']
+        else:
+            info['frame_div_parent'] = None
+    return new_info
+
+
+def snakecase_to_camelcase(name):
+    snake_pattern = re.compile(r'_([a-z])')
+    return snake_pattern.sub(lambda x: x.group(1).upper(), name)
```
Comment (willgraf) [inline deepcell_label/utils.py:27]: Might as well throw in a test for this too.

**Review by willgraf — CHANGES_REQUESTED** [review]
Looks awesome, a great improvement. In addition to a couple of small questions, it looks like there is some `etag` test failure going on.

File: visualizer/src/service/apiMachine.js (hunk 1 above)
Comment (tddough98) [reply visualizer/src/service/apiMachine.js:55]: Switched back to fetch

**Review by willgraf — APPROVED** [review]
💯

**Pull request merged**